event-processing/lambda/tests/unit/app.cleanser.test.ts

```diff
@@ -5,12 +5,45 @@ import { IEnrichedAuditEvent, IAuditEventUserMessage } from '../../models/enrich
 import { ICleansedEvent } from '../../models/cleansed-event';
 import { TestHelper } from '../test-helpers/test-helper';
 import { CleanserHelper } from '../test-helpers/cleanser-helper';
+import { mockClient } from 'aws-sdk-client-mock';
+import { GetSecretValueCommand, SecretsManagerClient } from '@aws-sdk/client-secrets-manager';
+import { ObfuscationService } from '../../services/obfuscation-service';
 
 describe('Unit test for app handler', function () {
     let consoleWarningMock: jest.SpyInstance;
+    const secretManagerMock = mockClient(SecretsManagerClient);
 
     beforeEach(() => {
         consoleWarningMock = jest.spyOn(global.console, 'log');
+        secretManagerMock.reset();
```

Comment:
Wonder if you could just mock out the KeyService (and add tests for it if there aren't any already). Since it's already in its own separate function, seems a shame not to take advantage of that fact to make the handler tests simpler?

Reply:
KeyService has been converted from a class to a getHmacKey function and mocked in these tests